programs/marinade-finance/src/instructions/admin/config_marinade.rs

```diff
@@ -156,6 +163,51 @@ impl<'info> ConfigMarinade<'info> {
                 None
             };
 
+        let withdraw_stake_account_enabled_change =
+            if let Some(withdraw_stake_account_enabled) = withdraw_stake_account_enabled {
+                let old = self.state.withdraw_stake_account_enabled;
+                self.state.withdraw_stake_account_enabled =withdraw_stake_account_enabled;
```

Comment:
Please use rust format to add a space after equal sign

Reply:
done